web-components/src/components/badge/scss/badge.scss

```diff
@@ -6,6 +6,11 @@
     @include badge-size;
     @include badge-style($radius: $badge--round__radius);
 
+    &:focus {
+      outline: 2px solid #007aa3;
```

Comment:
Use color token. and handle dark and light

Reply:
Done